### PR TITLE
[WIP] rewrite _request method, add tests

### PR DESCRIPTION
**Pull request by chinskiy**

### CODE REVIEW

File: pep8speaks/utils.py [hunk 1]
```diff
@@ -10,28 +10,25 @@
...
-        return requests.delete(*args, **kwargs)
+
+    kwargs = {
+        "auth": AUTH,
+        "json": json if json else None,
+        "data": data if data else None,
+        "headers": headers if headers else None,
+        "params": params if params else None,
+    }
+    return requests.request(method, query, **kwargs)
```
Comment (OrkoHunter) [inline pep8speaks/utils.py:31]: This is nice.

**Comment by OrkoHunter** [comment]
This looks cleaner ! Thank you.

**Comment by chinsky** [comment]
U're fast)
I was thinking to add more changes, but ok, will make another PR

**Comment by chinskiy** [comment]
U're fast)
I was thinking to add more changes, but ok, will make another PR

**Comment by OrkoHunter** [comment]
Oh sorry ! I overlooked the WIP tag the second time I saw the PR.

**Pull request merged**